editor/components/inner-blocks/index.js

```diff
@@ -43,15 +43,19 @@ class InnerBlocks extends Component {
 	updateNestedSettings() {
 		const {
 			blockListSettings,
-			allowedBlocks: nextAllowedBlocks,
+			allowedBlocks,
+			lock,
+			parentLock,
```

Comment:
Where is `parentLock` being passed?

Reply:
Sorry I missed the commit of parentLock retrieval during the rebase. It was corrected.